api.js

```diff
@@ -180,6 +186,25 @@ Api.prototype._prefixTitle = function (file) {
 	return prefix;
 };
 
+Api.prototype._restartTimer = debounce(function () {
```

Comment:
Ha of course. It implements the approach I was advocating. Missed it due to context switching :blush: